packages/flutter_tools/lib/src/windows/visual_studio.dart

```diff
@@ -312,6 +371,20 @@ class VisualStudio {
     return null;
   }
 
+  String? _findMsvcVersion(List<Map<String, dynamic>> installations) {
+    final String? installationPath = installations[0]['installationPath'] as String?;
+    String? msvcVersion;
+    if (installationPath != null) {
+      final Directory installationDir = _fileSystem.directory(installationPath);
+      final Directory msvcDir =
+          installationDir.childDirectory('VC').childDirectory('Tools').childDirectory('MSVC');
+      if (msvcDir.existsSync()) {
+        msvcVersion = msvcDir.listSync().whereType<Directory>().last.uri.pathSegments.where((String e) => e.isNotEmpty).last;
```

Comment:
Should we return null instead of throwing if there's no msvc version subdirectory?